docker/worker/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#set -e
+
+if [ -z $1 ]; then
+  echo "Error: define the number of workers"
+  echo "Usage: $0 <number-of-workers>"
+  exit
+fi
+
+for i in $(seq $1); do
+  docker run -d -h openqa_worker_$i --name openqa_worker_$i \
+  --volumes-from webui_data_1 \
+  --network webui_default \
+  -e OPENQA_WORKER_INSTANCE=$i \
+  --privileged openqa_worker_$(uname -m)
```

Comment:
I don't know any architecture containing white spaces but for the sake of being generic I'd add quotes here:
```suggestion
  --privileged "openqa_worker_$(uname -m)"
```

Reply:
I am removing this part. It is not necessarily applicable now,  but I guess will be important in the future. I think the docker builds are architecture dependent